index.html

```diff
@@ -8,6 +8,7 @@
     <link rel="stylesheet" href="index.css">
 </head>
 <body>
+    <!-- main container -->
     <div class="main_container">
         
```

Comment:
Please avoid using &emsp, instead using flex and grid in the layout to make it more responsive